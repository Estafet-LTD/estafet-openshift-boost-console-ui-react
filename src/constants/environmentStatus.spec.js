import ENVIRONMENT_STATUS from './environmentStatus';

describe('Constants: environmentStatus', () => {
  it('should match the snapshot', () => {
    expect(ENVIRONMENT_STATUS).toMatchSnapshot();
  });
});
