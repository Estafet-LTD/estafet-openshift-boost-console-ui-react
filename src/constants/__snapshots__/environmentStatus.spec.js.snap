// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: environmentStatus should match the snapshot 1`] = `
Object {
  "ACTIVE": "active",
  "TERMINATING": "terminating",
}
`;
